lib/mongodb.js

```diff
@@ -39,6 +40,7 @@ function ObjectID(id) {
   }
 }
 
+exports.generateMongoDBURL = generateMongoDBURL;
```

Comment:
They are both very useful in subclassing. If you look at the `exports.initialize` function it uses generateMongoDBURL. In our case we want to alter the main MongoDB function (to allow document versioning) through subclassing however the initialise function is basically unchanged except for invoking our new subclass instead of `new MongoDB`. 

Similarly `ObjectID` is useful to have access to when overriding the `coerceId` or `buildWhere` functions.